cmd/auth/main.go

```diff
@@ -24,16 +23,20 @@ import (
 	"github.com/mainflux/mainflux/auth/postgres"
 	"github.com/mainflux/mainflux/auth/tracing"
 	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/pkg/errors"
 	"github.com/mainflux/mainflux/pkg/uuid"
 	"github.com/opentracing/opentracing-go"
 	acl "github.com/ory/keto/proto/ory/keto/acl/v1alpha1"
 	stdprometheus "github.com/prometheus/client_golang/prometheus"
 	jconfig "github.com/uber/jaeger-client-go/config"
+	"golang.org/x/sync/errgroup"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 )
 
 const (
+	graceWaitTIme = 5
```

Comment:
Consider renaming variable. From current naming it looks like we are waiting to get samo grace, not waiting for a stop. Maybe `stopWaitTime` or `stopTimeout`? Also, take care of the letter case (`Time`, not `TIme`).